src/parser/regex/execRegex.ts

```diff
@@ -7,9 +7,12 @@ function execRegex(value: string, regex): string {
     if (REGEXES[regex]) {
       $regex = REGEXES[regex];
     }
-  } else {
+  } else if(regex.pattern) {
     const { pattern, flags } = regex;
     $regex = new RegExp(pattern, flags);
+
```

Comment:
In order to maintain the code standard in open source projects, it should be written in accordance with the rules of the project. For this, the project includes `eslint` and `prettier` files. Also, this project has a script called `lint` that will do this for you.

`yarn lint`